files/templates/volunteer_janitor.html

```diff
@@ -0,0 +1,60 @@
+{% macro display(duty) %}
+
+<div class="volunteer_janitor">
+    <div class="volunteer_text">
+        <p class="header">Janitor Duty</p>
+        <p class="text">Please rate these comments. You must choose a rating for each one. If you're undecided, just trust your gut!<p>
+        <p class="text"><a href="http://localhost/rules">Here's a link to the full rules</a> if you need a reference. Open this in a new tab; you may not be able to return to this page otherwise.</p>
+    </div>
+
+    <hr />
+
+    {% for c in duty.comments() %}
+
+        <div>
+            <div class="comment">
+                {% with comments=[c] %}
+                    {% include "comments.html" %}
+                {% endwith %}
+            </div>
+
+            <div class="choices">
+                <div>
```

Comment:
thought on looping through the enum instead?

Reply:
Yeah, I just wasn't sure how to do it and wanted to get it done :D

It's not just the enum, as such, it's also associated text labels.

If I ever find myself writing these names somewhere else I will absolutely de-duplicate my lookup table though.